src/main/java/com/pi4j/driver/display/BaseGraphicsDisplayComponent.java

```diff
@@ -0,0 +1,73 @@
+package com.pi4j.driver.display;
+
+public class BaseGraphicsDisplayComponent {
+
+    protected final GraphicsDisplayDriver driver;
+
+    public BaseGraphicsDisplayComponent(GraphicsDisplayDriver driver) {
+        this.driver = driver;
+    }
+
+    public void fill(int x, int y, int width, int height, java.awt.Color color) throws java.io.IOException {
```

Comment:
Maybe fillRect would be a better name?

please use int for color -- it's an established practice and avoids the awt dependency